webpack.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
     noParse: [/autoit.js/]
   },
   output: {
-    filename: './docs/bundle.js'
+    filename: '../docs/bundle.js'
```

Comment:
Do we need to fix this on the entry point as well?

Reply:
Actually, scratch this comment.  I get it now.  Output of webpack is the dist folder so `./` refers to dist.  Entry should be fine the way it is.